src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,97 @@
+        Scanner scanner = new Scanner(System.in);
+        User visitor = null;
+        System.out.println("Enter 'n' to create a new user or 'q' to quit application: ");
+        String command = checkValidStartCommand(scanner);
+        while (!command.equalsIgnoreCase(QUIT_APP)) {
+            if (!isValidCommand(command)) {
+                System.out.println("Unsupported command!");
+            }
+            if (command.equalsIgnoreCase(NEW_USER)) {
+                visitor = newVisitor(scanner);
+            }
+            if (visitor == null && command.equalsIgnoreCase(NEW_BET)) {
+                System.out.println("Minimum user information in appropriate format "
+                        + "must be provided to be sure you are allowed to make debts");
+            } else if (command.equalsIgnoreCase(NEW_BET) && visitor.isLegalGambling()) {
+                System.out.println(newBet(scanner));
+            }
+            System.out.println("\nEnter 'n' to create a new user, 'b' to make a bet "
+                                + "or 'q' to quit application: ");
+            command = scanner.nextLine();
+        }
+        System.out.println(userDao.getAllGamblers().toString());
+        System.out.println(betDao.getAll().toString());
+    }
+
+    private String checkValidStartCommand(Scanner scanner) {
+        String command = scanner.nextLine();
+        while (!(command.equalsIgnoreCase(NEW_USER) || command.equalsIgnoreCase(QUIT_APP))) {
+            System.out.println("Invalid option. Please, enter 'n' to create a new user "
+                                + "or 'q' to quit application: ");
+            command = scanner.nextLine();
+        }
+        return command;
+    }
+
+    private boolean isValidCommand(String command) {
+        return (command.equalsIgnoreCase(NEW_USER)
+                || command.equalsIgnoreCase(NEW_BET)
+                || command.equalsIgnoreCase(QUIT_APP));
+    }
+
+    private User newVisitor(Scanner scanner) {
+        System.out.println("Please, input some general information about you, "
+                            + "so we could accept and save your bets");
+        System.out.println("Enter your name and age");
+        String command = scanner.nextLine();
+        String[] userData = command.split(" ");
+        User visitor = null;
+        try {
+            String name = userData[0];
+            int age = Integer.parseInt(userData[1]);
+            visitor = new User(name, age);
+            userDao.saveAsGambler(visitor);
+        } catch (NumberFormatException | ArrayIndexOutOfBoundsException message) {
+            System.out.println("Please, input appropriate data");
+        }
+        System.out.println(visitor == null
+                ? "Unfortunately, provided information is not applicable for correct processing"
+                : "Your data has been successfully processed");
+        return visitor;
+    }
+
+    private String newBet(Scanner scanner) {
```

Comment:
```suggestion
    private String createNewBet(Scanner scanner) {
```
method names should be verbs